BushBlitz/ev-training-exercise
# Feature/evobe 106/create express js app

### Ticket
https://phunts.atlassian.net/browse/EVOBE-106
https://phunts.atlassian.net/browse/EVOBE-130
https://phunts.atlassian.net/browse/EVOBE-133

### Changes
- Setup of Express JS Project
- Established Folder Structure
- Added initial route file for `Employee` APIs
- Configured route crawling for all `.route.js` files inside `routes` folder

### Steps to Test
1. Pull branch
2. `npm install`
3. `npm start`
4. Go to `localhost:80/employee`
5. It should display `Employees` as a proof that the routing works

### Resources
None
